docs/dsp-tools-create-ontologies.md

```diff
@@ -755,6 +833,81 @@ This property is actually very similar to a simple text field.
 }
 ```
 
+
+#### isSequenceOf property
+
+`"object": (AudioRepresentation/MovingImageRepresentation or a subclass of one of them)`
+
+This property can be used, together with a `hasSequenceBounds` property, on a resource representing a sequence of an
+audio/video resource. The `isSequenceOf` would then point to the audio/video resource, and the `hasSequenceBounds` 
+would be the time interval of the sequence.
+
+The DSP base properties `isSequenceOf` and `hasSequenceBounds` can be used to derive a custom property from them, or 
+they can be used directly as cardinalities in a resource. The example below shows both possibilities.
+
+*gui-elements/gui_attributes*:
+
+- `Searchbox`: The only GUI element for _isSequenceOf_. Allows searching resources by entering the target resource name.
+    - _gui_attributes_:
+        - `numprops=integer` (optional): Number of search results to be displayed
+
+*Example:*
+
+```json
```

Comment:
not sure why Git Diff makes things in this file red? Maybe you can have a quick look to make sure there's nothing wrong here?

Reply:
found the culprit. The JSON in the example missed a closing bracket.